contentcuration/contentcuration/utils/celery/tasks.py

```diff
@@ -107,35 +124,56 @@ def shadow_name(self, *args, **kwargs):
         """
         return super(CeleryTask, self).shadow_name(*args, **kwargs)
 
-    def find_ids(self, channel_id=None, **kwargs):
+    def _prepare_kwargs(self, kwargs):
         """
-        :param channel_id:
-        :param kwargs: Keyword arguments sent to the task, which will be matched against
-        :return: A TaskResult queryset
-        :rtype: django.db.models.query.QuerySet
+        Prepares kwargs, converting UUID to their hex value
         """
-        task_qs = self.TaskModel.objects.filter(task_name=self.name)
+        return {
```

Comment:
Oh, that said - it seems like maybe doing an ordering here might be good?

Maybe here we could build an ordereddict instead of a regular dict to ensure key ordering?

```
return OrderedDict(
            (key, value.hex if isinstance(value, uuid.UUID) else value)
            for key, value in sorted(kwargs.items())
)
```
Or something like that.